contrib/inventory/ec2.py

```diff
@@ -647,6 +647,8 @@ def get_rds_instances_by_region(self, region):
                         instance.tags = {}
                         for tag in tags:
                             instance.tags[tag['Key']] = tag['Value']
+                        # collect the VPC ID from the DB subnet group
+                        instance.vpc_id = db_instances['DBInstances'][index]['DBSubnetGroup']['VpcId']
```

Comment:
odd - it's displayed when running the inventory for me. See the output in the ticket. Try disabling the cache (setting it to 0).